src/index.js

```diff
@@ -1,8 +1,9 @@
 // Helpers
 import 'react-app-polyfill/ie11';
 import 'react-app-polyfill/stable';
-import LogRocket from 'logrocket';
-import setupLogRocketReact from 'logrocket-react';
+import './assets/js/polyfills/details-element-polyfill';
+// import LogRocket from 'logrocket';
+// import setupLogRocketReact from 'logrocket-react';
```

Comment:
Still need a polyfill for includes:
![image](https://user-images.githubusercontent.com/13015201/102102495-c8e28d00-3e23-11eb-9ef9-b3e26daf6a4e.png)